app/__init__.py

```diff
@@ -85,9 +89,12 @@ def timeline():
 
 @app.route("/api/timeline_post", methods=["POST"])
 def post_time_line_post():
+    if "name" not in request.form: return "Invalid name", 400
     name = request.form["name"]
     email = request.form["email"]
+    if "@" not in email and ".com" not in email: return "Invalid email", 400
```

Comment:
although better methods prob exist for email checking i used this as a bandaid solution